data_acquisition/src/gee_functions.py

```diff
@@ -207,24 +207,23 @@ def ref_pull_457(image):
   # band where dswe is 3 and apply all masks
   dswe3 = d.eq(3).rename('dswe3').updateMask(f.eq(0)).updateMask(r.eq(1)).updateMask(s.eq(0)).selfMask()
   #calculate hillshade
-  h = CalcHillShades(image, wrs.geometry()).select('hillShade')
+  h = calc_hill_shades(image, wrs.geometry()).select('hillShade')
   #calculate hillshadow
-  hs = CalcHillShadows(image, wrs.geometry()).select('hillShadow')
+  hs = calc_hill_shadows(image, wrs.geometry()).select('hillShadow')
   
-  pixOut = (# make some dummy bands for the reducer
-            image.select(['Blue', 'Green', 'Red', 'Nir', 'Swir1', 'Swir2', 'SurfaceTemp', 'temp_qa'],
+  pixOut = (image.select(['Blue', 'Green', 'Red', 'Nir', 'Swir1', 'Swir2', 'SurfaceTemp', 'temp_qa'],
             ['med_Blue', 'med_Green', 'med_Red', 'med_Nir', 'med_Swir1', 'med_Swir2', 'med_SurfaceTemp', 'med_temp_qa'])
-            .addBands(image.select(['Blue', 'Green', 'Red', 'Nir', 'Swir1', 'Swir2', 'SurfaceTemp', 'temp_qa', 'ST_CDIST'],
```

Comment:
removal of bands that we don't ever use due to the LS7 correction. if we were using data from only one satellite, we could also export these and they would be meaningful, but it's just junk that we don't look at, so removing for now.